tests/unit/model.py

```diff
@@ -23,7 +23,6 @@
 
 from haas.model import *
 from haas import config
-from haas.ext.obm.ipmi import Ipmi
```

Comment:
is there a way we could still import this once and have it be used by the Nic, Node, and NetworkingAction tests?